services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ConnectedDevicesSpec.groovy

```diff
@@ -705,7 +886,9 @@ srcLldpDevices=#newSrcEnabled, dstLldpDevices=#newDstEnabled"() {
     private void enableMultiTableIfNeeded(boolean lldpEnabled, SwitchId switchId) {
```

Comment:
what about adding additional parameters (e.g initSwProps) ?
then you can just do:
```
!initSwProps.multiTable && northbound.updateSwitchProperties(sw.dpId, initSwProps.jacksonCopy().tap {
            it.multiTable = true
        })
```
What I mean is you can avoid sending needless get request `northbound.getSwitchProperties(sw.dpId)`

Reply:
done